### ISSUE TITLE
Mp4 opens in Firefox as opposed to downloading

### ISSUE THREAD

**Issue by rolandixor** [YePpHa/YouTubeCenter#2307]
The download feature is not working correctly. Instead of downloading, the files are being opened in a new tab and played.

**Comment by Micksy** [comment]
Yep, since the start of this week, developer version 538 in greasemonkey 3.9 on Firefox 49.0.2.

HD 1080p download mp4 works, but trying to grab a 360p version opens a 2nd instance of the file.

**Comment by Micksy** [comment]
Hey @rolandixor 

I had the same issue, found a workaround in Firefox.

Go to the about:config page.

Type in *.mp4 in the search box.

change
"media.mp4.enabled" to false.

restart Firefox.

So far I've not seen any issues.

**Comment by ulkoalex** [comment]
> It names all files to "videoplayback" without a file extension.
Can confirm. Something with youtube servers itself

**Comment by digideth** [comment]
@Micksy That would be a nice hack for the moment but I don't even have "media.mp4.enabled" listed in my about:config

For me the Only download that works correctly its the 720p mp4. Everything else opens a tab.
Considering this issues has be around for almost 2 months and the problem, i think, is on youtubes side I don't think it will be fixed. :(

**Comment by saero** [comment]
Man, this is really annoying. YT-C keeps getting broken all the time, thanks Google.

**Comment by ROMaster2** [comment]
A temporary work around would be to right-click > Save Link As... or copy the link and open the video in a media player like VLC or MPC-HC. A permanent solution is preferred.

**Comment by rolandixor** [comment]
The permanent solution is for someone who is proficient to fork this addon and create a new one that does the same job. Clearly the developer has abandoned development for whatever reason.
